ddtrace/internal/ci_visibility/writer.py

```diff
@@ -106,11 +106,11 @@ def __init__(
             timeout = config._agent_timeout_seconds
         intake_cov_url = None
         if use_evp:
-            intake_url = agent.get_trace_url()
-            intake_cov_url = agent.get_trace_url()
+            intake_url = intake_url if intake_url else agent.get_trace_url()
+            intake_cov_url = intake_url
```

Comment:
I like it this way because it makes it clear that we want to use the same URL as the `intake_url` when we have an agent, or when we're hardcoding the agentless URL.